bindings/python/common.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include <string>
+
+#include <pire/stub/noncopyable.h>
+#include <pire/pire.h>
+
+#ifndef PIRE_NO_CONFIG
+#   include <pire/config.h>
+#endif
+
+#if ! defined(SWIG) && defined(HAVE_RVALUE_REFERENCES)
+#   define PIRE_SWIG_RVALUE_REF &&
+#else
+#   define PIRE_SWIG_RVALUE_REF
+#endif
+
+typedef const std::string& string_view;
```

Comment:
The intended semantics of my `string_view` is actually similar, although I should probably rename it to `StringView`.